agent/stats/engine.go

```diff
@@ -40,9 +40,20 @@ const (
 	queueResetThreshold    = 2 * ecsengine.StatsInactivityTimeout
 )
 
+// statsEngineMapType is an enumeration of the map type in stats engine
+type statsEngineMapType int
+
+const (
+	// statsMetricsMap represents the map 'tasksToContainers'
+	statsMetricsMap statsEngineMapType = iota
+	// statsHealthmetricsMap represents the map 'tasksToHealthCheckContainers'
+	statsHealthmetricsMap
+)
+
 // EmptyMetricsError indicates an error for a task when there are no container
 // metrics to report
 var EmptyMetricsError = errors.New("stats engine: no task metrics to report")
+var EmptyHealthMetricsError = errors.New("stats engine: no task health metrics to report")
```

Comment:
missing `// EmptyHealth...` here. Also, you can do this to organize this better:

```
var (
 EmptyMetricsError = ...
 EmptyHealthMetricsError = ...
)
```